### DISCUSSION TITLE
How can I generate images I choose,not just a sample

### DISCUSSION THREAD

**Discussion by aaawrong** [open-mmlab/mmgeneration#459] [How-to] [answered]
If I want to use CelebA to generate 202599 fake images and keep them in a flie  ,how should I do ?

**Comment by plyfager** [comment]
Hi @DCMJY , take `configs/pggan/pggan_celeba-cropped_128_g8_12Mimgs.py` as an example,
 delete its `metrics` field and run
```
bash tools/eval.sh configs/pggan/pggan_celeba-cropped_128_g8_12Mimgs.py CKPT_PATH --samples-path PATH_TO_SAVE_IMGS --num-samples 202599
```
Then you will find images under `PATH_TO_SAVE_IMGS`.

**Comment by plyfager** [comment]
@LeoXing1996 Sampling images this way seems not convenient for users. Maybe we can modify `xxx_demo.py` for this usage.

**Reply by bsssss1** [reply]
Hi@**plyfager**,how to fix my problem which is some of my generated fake images look like similarity?
![2023-03-18 15-15-03 的屏幕截图](https://user-images.githubusercontent.com/83162328/226091258-033d6721-b995-40a7-a2d3-fe75f40bdfb5.png)

**Comment by aaawrong** [comment, marked as answer]
> Hi @DCMJY , take `configs/pggan/pggan_celeba-cropped_128_g8_12Mimgs.py` as an example, delete its `metrics` field and run
> 
> ```
> bash tools/eval.sh configs/pggan/pggan_celeba-cropped_128_g8_12Mimgs.py CKPT_PATH --samples-path PATH_TO_SAVE_IMGS --num-samples 202599
> ```
> 
> Then you will find images under `PATH_TO_SAVE_IMGS`.

Thank you !!!
